channeldb/invoices.go

```diff
@@ -1443,10 +1481,16 @@ func copyInvoice(src *Invoice) *Invoice {
 		Htlcs: make(
 			map[CircuitKey]*InvoiceHTLC, len(src.Htlcs),
 		),
+		HodlInvoice: src.HodlInvoice,
```

Comment:
do we still want to create `.Copy()` methods on `Invoice` and `InvoiceHTLC`? can be done later too, just curious

Reply:
I prefer later because this PR is sort of on the critical path (at least in the vicinity of the critical path).